ReactWindows/ReactNative.Shared/UIManager/Events/TouchEventTypeExtensions.cs

```diff
@@ -25,6 +25,8 @@ public static string GetJavaScriptEventName(this TouchEventType eventType)
                     return "topMouseOver";
                 case TouchEventType.Exited:
                     return "topMouseOut";
+                case TouchEventType.PointerMove:
+                    return "topMouseMoveCustom"; // Using a non-clashing name until this one propagates: https://github.com/facebook/react/commit/e96dc140599363029bd05565d58bcd4a432db370
```

Comment:
Basically we can change this once we update to `react-native@0.57` (and we have `react@16.4+`)